llvm/lib/Target/RISCV/MCTargetDesc/RISCVInstPrinter.cpp

```diff
@@ -269,7 +293,7 @@ void RISCVInstPrinter::printVMaskReg(const MCInst *MI, unsigned OpNo,
     return;
   O << ", ";
   printRegName(O, MO.getReg());
-  O << ".t";
+  markup(O, Markup::Register) << ".t";
```

Comment:
Highlighting the two parts of the register separately with a reset and re-colour in the middle seems a bit ugly